tests/search_derived/mobile_search_clients_daily_v1/test_overactive_filter/org_mozilla_firefox.metrics.yaml

```diff
@@ -0,0 +1,2 @@
+---
```

Comment:
What's going on with these files?

Reply:
These are just empty arrays so that the test script will create empty tables.  Otherwise it will error with dataset/table not found